### PR TITLE
clang-tidy: Identifier rename IfdsIde FF/EF

### PR DESCRIPTION
**Pull request by MMory**
~~please merge after #452~~

### CODE REVIEW

File: include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/DefaultSeeds.h [hunk 1]
```diff
@@ -25,12 +25,12 @@ namespace psr {
...
 template <typename N, typename D> class DefaultSeeds {
 public:
-  static std::map<N, std::set<D>> make(std::vector<N> node, D zeroNode) {
-    std::map<N, std::set<D>> res;
-    for (N n : node) {
-      res.insert(n, std::set<D>{zeroNode});
+  static std::map<N, std::set<D>> make(std::vector<N> Nodes, D ZeroNode) {
+    std::map<N, std::set<D>> Result;
+    for (N Node : Nodes) {
+      Result.insert(Node, {ZeroNode});
```
Comment (vulder) [inline include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/DefaultSeeds.h:31]: As we have copied the `Nodes` vector already, we should move the elements into the map.
```suggestion
      Result.insert(std::move(Node), {ZeroNode});
```

File: include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposer.h [hunk 2]
```diff
@@ -39,8 +39,8 @@ class EdgeFunctionComposer
 
 private:
   // For debug purpose only
-  const unsigned EFComposer_Id;
-  static inline unsigned CurrEFComposer_Id = 0;
+  const unsigned EFComposerId;
+  static inline unsigned CurrEFComposerId = 0; // NOLINT
```
Comment (vulder) [inline include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposer.h:43]: What's the issue here?

File: unittests/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposerTest.cpp [hunk 3]
```diff
@@ -16,9 +16,9 @@ static unsigned CurrAddTwoEfId = 0;
 struct MyEFC : EdgeFunctionComposer<int> {
   MyEFC(std::shared_ptr<EdgeFunction<int>> F,
         std::shared_ptr<EdgeFunction<int>> G)
-      : EdgeFunctionComposer<int>(std::move(F), std::move(G)){};
+      : EdgeFunctionComposer<int>(F, G){};
   std::shared_ptr<EdgeFunction<int>>
```
Comment (vulder) [inline unittests/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposerTest.cpp:20]:
```suggestion
      : EdgeFunctionComposer<int>(F, G){};

  std::shared_ptr<EdgeFunction<int>>
```
An empty line here would make it more readable ;)

PS: Why did you remove the `moves`?

**Review by vulder — CHANGES_REQUESTED** [review]

**Review by pdschubert — APPROVED** [review]

File: include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposer.h (hunk 2 above)
Comment (pdschubert) [reply include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposer.h:43]: `Variable 'CurrEfComposerId' is non-const and globally accessible, consider making it constclang-tidy(cppcoreguidelines-avoid-non-const-global-variables)`
